charmtools/build/builder.py

```diff
@@ -882,8 +882,14 @@ def main(args=None):
                         action="store_true")
     parser.add_argument('--interface-service',
                         help="Deprecated: use --layer-index")
-    parser.add_argument('--layer-index',
-                        default=LayerFetcher.LAYER_INDEX)
+    parser.add_argument('-i', '--layer-index',
+                        help='One or more index URLs use to look up layers, '
+                             'separated by commas. Can include the token '
+                             'DEFAULT, which will be replaced by the default '
+                             'index{}: {}'.format(
```

Comment:
It would be nice to 1) show an example of what this looks like (e.g. https://mysite.com/my-index/,DEFAULT), and also show what the default index is (same for the help on pullsource.py).

Reply:
I changed my mind about this, I think what's here is fine.

Reply:
The default value is shown (the join bit).  I'll add the example.